src/test/java/kostka/moviecatalog/service/CacheServiceTest.java

```diff
@@ -29,47 +31,47 @@
     @InjectMocks
     CacheService cacheService;
 
-    private Generator generator = new Generator();
+    private ObjectMapper mapper = new ObjectMapper();
 
     @Test
-    public void getAllMoviesFromRedisCacheTest() {
-        Movie movie = generator.createMovieWithName(TEST_NAME);
-        Movie movie2 = generator.createMovieWithName(TEST_NAME_2);
-        List<Movie> movies = Arrays.asList(movie, movie2);
-
-        Mockito.when(redisService.getMoviesWithKey(ALL_MOVIES_KEY)).thenReturn(movies.toString());
+    public void getAllMoviesFromRedisCacheTest() throws JsonProcessingException {
+        String moviesJson = createMoviesJson();
+        Mockito.when(redisService.getMoviesWithKey(ALL_MOVIES_KEY)).thenReturn(moviesJson);
```

Comment:
I am confused here. What are you testing here?

Reply:
You configure mockito to return certain value, then retrieve it and make assertion.
Are you making sure Mockito works as expected? :)

Reply:
good point, as the cache service has changed in the past, I would need to test cacheservice instead of restservice :) thanks!